package.json

```diff
@@ -50,8 +50,10 @@
     "html"
   ],
   "devDependencies": {
-    "@markedjs/testutils": "12.0.0-0",
     "@arethetypeswrong/cli": "^0.15.1",
+    "@babel/plugin-transform-unicode-regex": "^7.23.3",
+    "@markedjs/testutils": "12.0.0-0",
+    "@rollup/plugin-babel": "^6.0.4",
```

Comment:
We don't want to use babel for the output. This will make marked a lot slower and a lot bigger. If users need to support older browsers they can use babel themselves.

Reply:
Actually I think babel here is just to transform the unicode regex, won't do other stuff 🤔